spec/routing/api/v1/keywords_routing_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::KeywordsController, type: :routing do
+  describe 'routing' do
+    it { expect(get: 'api/v1/keywords').to route_to('api/v1/keywords#index') }
```

Comment:
we usually define the test clearer in this case, e.g.
```ruby
it 'routes /keywords to keyword controller' do
  expect(get: 'api/v1/keywords').to route_to('api/v1/keywords#index')
end
```

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/17 and https://github.com/gutakk/ruby-google-scraper/pull/19